composer.json

```diff
@@ -56,21 +56,23 @@
       "type": "package",
       "package": {
         "name": "alleyinteractive/searchpress",
-        "version": "0.4.1",
+        "version": "dev-main",
         "type": "wordpress-plugin",
         "source": {
           "url": "https://github.com/alleyinteractive/searchpress.git",
           "type": "git",
-          "reference": "v0.4.1"
+          "reference": "main"
         }
       }
     }
   ],
+  "require": {
+    "php": "^8.1 || ^8.2"
+  },
   "require-dev": {
-    "php": "^7.4 || ^8.0",
     "alleyinteractive/alley-coding-standards": "^0.3.0",
-    "alleyinteractive/pest-plugin-wordpress": "^0.4.0",
-    "alleyinteractive/searchpress": "^0.4.1",
+    "alleyinteractive/pest-plugin-wordpress": "dev-feature/php-8-1-upgrade",
```

Comment:
Suggestion to update to the latest version: https://github.com/alleyinteractive/pest-plugin-wordpress/releases/tag/v0.5.0

Reply:
@renatonascalves ty! can you give this another look, please?